projects/packages/jetpack-mu-wpcom/src/features/wpcom-options-general/options-general.php

```diff
@@ -50,8 +50,8 @@ function wpcom_site_management_panel_link() {
  */
 function wpcom_site_management_panel() {
 	$current_screen = wpcom_admin_get_current_screen();
-	if ( in_array( $current_screen, WPCOM_DUPLICATED_VIEW, true ) && wpcom_is_duplicate_views_experiment_enabled() ) {
-		add_settings_field( 'wpcom_site_management_panel_link', __( 'WordPress.com Site Settings', 'jetpack-mu-wpcom' ), 'wpcom_site_management_panel_link', 'general', 'default' );
+	if ( in_array( $current_screen, WPCOM_DUPLICATED_VIEW, true ) ) {
+		add_settings_field( 'wpcom_site_management_panel_link', __( 'WordPress.com Site Settings', 'jetpack-mu-wpcom' ), 'wpcom_site_management_panel_link', 'general' );
```

Comment:
Fifth parameter ($section) dropped because we were setting it to `default`, which is the same behaviour as when omitting that parameter.